tests/harness/close.rs

```diff
@@ -0,0 +1,47 @@
+use bitcoin::Address;
+use thor::{close, close::FinalState, Channel};
+
+pub struct Init {
+    pub alice: Channel,
+    pub bob: Channel,
+}
+
+impl Init {
+    pub fn new(alice: thor::create::Party6, bob: thor::create::Party6) -> Self {
+        let alice = Channel::new(alice);
+        let bob = Channel::new(bob);
+
+        Self { alice, bob }
+    }
+}
+pub struct Final {
```

Comment:
Back to an argument I made to @luckysori in another PR, if the first and only thing you do with a struct is deconstructing it (which is the case here) then it means a tuple is enough :)

Reply:
> if the first and only thing you do with a struct is deconstructing it (which is the case here) then it means a tuple is enough

I don't necessarily agree with this. Sometimes it's enough, but the names provided by a dedicated return type can help with clarity. Particularly if you're returning two (or more) of the same type.

Reply:
Well, but there we are not returning more than two and same type variables.

Reply:
Yep, we agree then.